src/imfill.jl

```diff
@@ -0,0 +1,47 @@
+"""
+ Image filling Algorithm
+
+ ```
+ filled_img = imfill(img, min_size)
+ filled_img = imfill(img, min_size, value)
+ filled_img = imfill(img, min_size, value, connectivity)
+ ```
+
+ Connected components of an image is found using flood-fill algorithm and returns a copy of
+ the original image after filling objects that are smaller than the specified min_size.
+
+
+ Parameters:
+
+  -  img            = Input image (Boolean array type)
+  -  min_size       = Minimum size of objects that are not filled.
```

Comment:
I would add an option for `max_size` as well, for filling regions above a threshold, would that be useful? This would be more flexible, and I think the implementation wouldn't change much. Please let me know what you think.

I would actually replace the option `min_size` by an interval tuple `interval=(-Inf,Inf)` within which regions and holes would be filled if their sizes lie in the interval.

Reply:
@juliohm I added the interval option and the algorithm seems to be working fine.
Also, I added a default range inspired from **scikit-image** `remove_small_holes` so that the algo will handle small holes and also work for larger region on user input.